triggers/service/BUILD.bazel

```diff
@@ -239,14 +240,12 @@ da_scala_test_suite(
     deps = test_deps,
 )
 
-da_scala_test_suite(
+# The "manual" test tag doesn't seem to work with `da_scala_test_suite`.
+da_scala_test(
```

Comment:
there are a bunch of unused deps here. Not caused by your PR but `da_scala_test_suite` (deliberately) disables unused dep checks so now it blows up. Should be able to just kill them.

Reply:
Oh, missed those somehow. I purged the ones in participant-integration-api.

Not sure how `bazel build //...` didn't catch that.

Reply:
Seems I needed to run `DAML_ORACLE_TESTING=true bazel build //...`, even with the change to the manual tags. I guess they don't get built otherwise.